src/client/pages/Debugger/components/Offer.tsx

```diff
@@ -94,6 +164,7 @@ export const Offer: React.FC = () => {
             {() => (
               <InputField
                 label="Quote id"
+                name="Quote id"
```

Comment:
Now that we generate the uuid, we might as well set it the input to disabled since there's no need to edit it right?
```suggestion
                name="Quote id"
                disabled
```

Reply:
But we want to be able to paste in an existing id and get the quote maybe?

Reply:
Is that a reason to leave it as is @gustaveen ? What do you think @palmenhq ?

Reply:
But right now we can't fetch an already created quote right (and I don't think we need it). If you edit the id you will end up creating a new quote

Reply:
Well actually 🤓 If you paste an id of a quote you already created in the quote id field the quote is fetched and the data for that quote is displayed. Since the fetching is triggered by change to the quote id. So we can fetch already created quotes! 
I don't see the use case for it either though, but I have a faint memory of @palmenhq saying that was the idea behind having an input field for the id in the first place.

Reply:
OK, since we agreed on keeping this as it is (for now, maybe adding a button to generate a UUID in the future) I will resolve this conversation! 😇